tools/packrom/packrom.cpp

```diff
@@ -67,9 +73,10 @@ typedef struct{
 	u8 author[32];
 	u8 icon[16*16];
 	u32 crc32;
-	u8 mouse;
+	u8 psupport; //peripherals supported
 	u8 description[64];
-    u8 reserved[114];
+	u8 pdefault; //peripherals to enable by default(Emulator)
+	u8 reserved[113];
 }RomHeader;
```

Comment:
Please add a comment here that if this struct is modified, that it also needs to be modified in uzerom.h